openfisca_survey_manager/scenarios.py

```diff
@@ -754,7 +754,7 @@ def init_from_data(self, calibration_kwargs = None, inflation_kwargs = None,
 
         :param data:                Contains the data, or metadata needed to know where to find it.
 
-        :param use_marginal_tax_rate: True to go into marginal effective tax rate computation mode
+        :param use_marginal_tax_rate: Bool, True to go into marginal effective tax rate computation mode
```

Comment:
You should follow http://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html

Reply:
Actually I changed it a bit and commited.